File: EQIS_env-main/react_sample/react/src/interfaces/commonCode/commonCodeListSearch.ts
import { cloneDeep } from 'lodash';

import { baseInit, Ibase } from '@/interfaces/common/base';

export interface IcommonCodeListSearch extends Ibase {
    comCdGrpCd: string;
    comCd: string;
    langCd: string;
    langComCdNm: string;
}

export const commonCodeListSearchInit = () => {
    const commonCodeListSearch: IcommonCodeListSearch = {
        comCdGrpCd: '',
        comCd: '',
        langCd: '',
        langComCdNm: '',
        ...baseInit(),
    };
    return cloneDeep(commonCodeListSearch);
};

File: EQIS_env-main/react_sample/springboot/Spring/src/main/java/com/sample/basic/sample/code/model/CommonCodeListSearch.java
package com.sample.basic.sample.code.model;

import com.sample.basic.cmm.model.Base;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@Data
@EqualsAndHashCode(callSuper=false)
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class CommonCodeListSearch extends Base {

	private String comCdGrpCd;
	private String comCd;
	private String langCd;
	private String langComCdNm;
	
}
